challenge-05/js/script.js

```diff
@@ -0,0 +1,27 @@
+var arr = [
+    "http://via.placeholder.com/350x150?text=1",
+    "http://via.placeholder.com/350x150?text=2",
+    "http://via.placeholder.com/350x150?text=3"
+];
+var container = document.querySelector('img');
+
```

Comment:
@vikrep Well done, the main functionalities are working but the code could be improved by:
- var `arr` not needed  as you can use query selector to get the number of images.
- the functionaries can be split into smaller function, e.g selectActiveImages, disableButton, ScollTo .... as i mentioned above.